plugins/Transitions/javascripts/transitions.js

```diff
@@ -1586,8 +1586,8 @@ Piwik_Transitions_Ajax.prototype.callApi = function (method, params, callback) {
 
                 if (typeof Piwik_Transitions_Translations == 'undefined') {
                     self.callApi('Transitions.getTranslations', {}, function (response) {
-                        if (typeof response[0] == 'object') {
-                            Piwik_Transitions_Translations = response[0];
+                        if (typeof response == 'object') {
+                            Piwik_Transitions_Translations = response;
```

Comment:
Why was this change required? Wasn't that working correctly before?

Reply:
This wasn't working and the translations were not being loaded. The API returns an object with the translation strings but the response handler is checking for a sub-object for some reason.

Reply:
Probably related to a change a while back where we removed the wrapping array from json responses?

See https://github.com/matomo-org/matomo/commit/66bcb98556e5b92b7b6187a5a904e13a408d0ed6#diff-287d49d80349087bbbaa1186c9f825689026f878c9bad60f46a735b65011acb2R64

and https://github.com/matomo-org/matomo/pull/18136